VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioEditorModule.cpp

```diff
@@ -152,6 +172,15 @@ TSharedRef<FExtender> ExtendLevelViewportContextMenuForVitruvioComponents(const
 					FSlateIcon(), AddVitruvioComponentAction);
 			}
 
+			if (HasAnyVitruvioActor(SelectedActors))
+			{
+				const FUIAction CookVitruvioActorsAction(FExecuteAction::CreateStatic(CookVitruvioActors, SelectedActors));
+
+				MenuBuilder.AddMenuEntry(FText::FromString("Cook Vitruvio Actor"),
```

Comment:
We should maybe choose a name which is a bit easier to understand (e.g. "Convert to static mesh)